Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent any 
+
+    environment {
+            PIPELINE_JOBS_NAME = 'edx-app-android-pipeline'
+            ANDROID_HOME = '/opt/android-sdk-linux'
+            WORKSPACE = "/var/lib/jenkins/workspace/$PIPELINE_JOBS_NAME"            
```

Comment:
No need to explicitly set this. $WORKSPACE is already a variable in Jenkins (see https://wiki.jenkins.io/display/JENKINS/Building+a+software+project)

Reply:
done, thanks.